menu/menu_setting.c

```diff
@@ -6640,7 +6640,8 @@ void general_write_handler(rarch_setting_t *setting)
          }
          break;
       case MENU_ENUM_LABEL_VIDEO_SMOOTH:
-         video_driver_set_filtering(1, settings->bools.video_smooth);
+      case MENU_ENUM_LABEL_VIDEO_HW_SCALING:
+         video_driver_set_filtering(1, settings->bools.video_hw_scaling, settings->bools.video_hw_scaling);
```

Comment:
The `settings->bools.video_smooth` setting is now completely ignored. Would it be better to just re-purpose that setting?

Reply:
My first proof of concept indeed used used hw scaling when video_smooth was false but there are some cases of my testers wanting non-hw scaling (which implies bicubic).